### ISSUE TITLE
put upscaling in its own command

### ISSUE THREAD

**Issue by pachacamac** [brycedrennan/imaginAIry#99]
Is there a way to perform only upscaling without any other generation steps?

I tried for example `imagine --init-image pic.jpg --mask-prompt all --mask-mode keep --upscale --fix-faces ""` so using an empty prompt but it still completely changes the image. Same when not using `--fix-faces`

**Comment by brycedrennan** [comment]
I should make a command to do that directly but for now you could do:

```
imagine --init-image img.jpg --init-image-strength 1 --upscale ""
```

**Comment by Titaniumtown** [comment]
any progress on this being a dedicated command?

**Comment by brycedrennan** [comment]
Added in just released 8.3.0

**Comment by maschinenzeitmaschine** [comment]
if i understand these two lines in the readme correctly, this now exists, doesn't it?
```
# upscale an existing image
>> aimg upscale my-image.jpg

```
but… how would i use this from python? is there any example somewhere? i'm really struggling with documentation in general, am i missing something? (apart from the readme which is, apart from a very short section, only about command line usage...)
